### DISCUSSION TITLE
How to export a flextable as html in a shiny app?

### DISCUSSION THREAD

**Discussion by acafonsosilva** [davidgohel/flextable#574] [Q&A]
I am trying to make a simple shiny app to create a flextable that can be exported as HTML using save_as_html and the downloadHandler button but it's not working since it just gives me an empty HTML file. In my R script before building a shiny app, I was able to export a flextable with save_as_html and then use bookdown::chrome_print to get a pdf file, but I am not even able to first get the HTML file from within shiny (even while using htmltools_value()). 

I'm not sure if it helps, but from another shiny app using the gt package, I could export the file with gtsave.

Any suggestion of how I could go about this?
Thanks!

**Comment by davidgohel** [comment]
With no reproducible example, it's not possible to help.
